.github/workflows/npm-audit.yml

```diff
@@ -0,0 +1,20 @@
+name: Npm Audit
+
+on: [push, pull_request]
+
+jobs:
+  npm-audit:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+    - name: Set up Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '20'
+    - name: Remove @triggi/ dependencies
+      run: sed '/"@triggi\/[A-Za-z0-9-]\+" \?:/d' package.json --in-place
```

Comment:
This will leave a syntactically incorrect file for cases where the @triggi lines were the last in the list and not also the first. I think it would solve it if it would also remove any ',' before the line it's removing, if present.